### PR TITLE
[COM-346] Update README

### PR DESCRIPTION
**Pull request by Spuffynism**
- Point doc to 2.0
- Explain what the project is about.
- Explain setup steps
- Explain how to run tests
- Explain how to contribute in a “CONTRIBUTIONS.md” file
- Demonstrate a page view use case (Some were already present, should we add more detailed ones?)

Will squash commits on merge!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,43 +61,42 @@ The code snippet must contain `contentIdKey` and `contentIdValue` in order to id
 | contentIdValue | The Coveo index field value that will be used to identify the item. |
 | contentType    | [Optional] The type of the item to be tracked (e.g., 'Article').    |
 
-**Note: Do not copy the\_ `coveoua.js` \_file as it can be updated anytime and you could experience compatibility issues.**
+**Note: Do not copy the `coveoua.js` file as it can be updated anytime and you could experience compatibility issues.**
 
-3. Validate pageview events are pushed to the Coveo Usage Analytics service
+3. Validate `pageview` events are sent to the Coveo Usage Analytics service
 
 a. In a web browser such as Chrome, navigate to a website page to which you added the code snippet.
```
[the remark this answers is not in the document]
Comment (agagnon-coveo) [reply README.md:68]: This is a good example of what my comment above is describing I think.

**Review by agagnon-coveo — COMMENTED** [review]
Solid cleanup :muscle:

File: README.md [hunk 2]
```diff
@@ -8,53 +8,51 @@
...
 
-Initially, the `pageview` events data will be used exclusively by the Coveo Reveal Recommendations feature (see [Recommendations Feature](https://onlinehelp.coveo.com/en/cloud/coveo_reveal_features.htm#Recommendations)). It is recommended that you start pushing `pageview` events to the Coveo Usage Analytics service as soon as possible so that you get relevant items recommended.
+Initially, the `pageview` events data will be used exclusively by the Coveo ML Event Recommendations (ER) Feature (see [Event Recommendations (ER) Feature](https://docs.coveo.com/en/1671/coveo-machine-learning/coveo-machine-learning-features#ER)). It is recommended that you start sending `pageview` events to the Coveo Usage Analytics service as soon as possible so that you get relevant items recommended.
+
+On top of `pageview` events, generic and commerce (see [Tracking Commerce Events](https://docs.coveo.com/en/3188/coveo-solutions/tracking-commerce-events)) events can also be tracked.
```
Comment (francoislg) [inline README.md:22]: The "see Tracking Commerce Events" is a little awkward in the middle of the sentence, you could put it at the end?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Spuffynism) [reply README.md:68]: Thanks! changing this.

**Review by cnguyen-coveo — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -8,53 +8,51 @@
...
 
-### Pushing Coveo Analytics Pageview Events for Recommendations
+### Sending Coveo Analytics Page View Events for Recommendations
 
 1. Get an API key.
 
-You need a Coveo Cloud API key that has the permission to write to the Usage Analytics service.
-
--   For [Coveo Cloud V1](https://cloud.coveo.com/), contact [Coveo Support](https://coveocommunity.force.com/) and ask to create an API key with the **Write UA** scope.
+You need a Coveo Cloud API key which has the [**Push** access level on the **Analytics Data** domain](https://docs.coveo.com/en/1707/cloud-v2-administrators/privilege-reference#analytics-data-domain) to send events (see [User Authentication](https://docs.coveo.com/en/3188/coveo-solutions/tracking-commerce-events#user-authentication)).
```
[the remark this answers is not in the document]
Comment (Spuffynism) [reply README.md]: 100% agree! I applied these changes. :smile:

**Review by francoislg — APPROVED** [review]
Looks very good! Great job!

It could be done later on (or even in a completely different story), but I think it is still missing some documentation details about the Tracking Commerce Events since this is its most useful use case. 

It does not have to be as exhaustive as our doc, but just an example to show that it is possible would steer people towards the right thing :)

File: README.md (hunk 2 above)
Comment (Spuffynism) [reply README.md:22]: Yeah sure!

I'll also add a "Sending Commerce Events" section after the "Sending Any Coveo Analytics Event" section!

**Review by agagnon-coveo — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -8,98 +8,129 @@
...
 You can now call the script using the following line:
 
-```html
+```js
 coveoua('send', 'click', { ... });
 ```
 
 Refer to the [Usage Analytics Write API](https://docs.coveo.com/en/1430/cloud-v2-developers/usage-analytics-write-api) section to see what types of events are supported and what payload you need to send.
 
+### Sending Commerce Events
```
Comment (Spuffynism) [inline README.md:97]: @francoislg Let me know what you think of this section. 

Lines 114 to 125 are almost a direct copy of the documentation on tracking addition to cart interactions (https://docs.coveo.com/en/3188/coveo-solutions/tracking-commerce-events#measuring-an-addition-to-the-cart).

**Review by aturar-coveo — APPROVED** [review]

**Pull request merged**